src/js/modules/rpc/server.ts

```diff
@@ -223,9 +223,10 @@ export class ProcessBatchServer extends SupervisorServer {
     const handle = module?.exports?.default;
     if (handle === undefined) {
       this.logger.error(
-        "Error on load script, script doesn't " + "export anything"
+        "Error on load script, script doesn't export anything. " +
+          "Possible cause: script not packaged using NPM."
       );
-      return undefined;
+      return [undefined, errors.validateLoadScriptError(null, id, script)];
```

Comment:
```
Fix wasm crash bug when deploy unpackaged scripts.
```

When writing commit message please write a sentence or two to describe what the problem is and how the commit fixes the problem. It can be short and simple for easy things, or as long as necessary to describe things that aren't obvious and/or complicated.

Reply:
Just updated commit message:)